src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -91,18 +87,26 @@ public void generateReport(
           throw new RuntimeException("Couldn't find resource header");
         }
         for (String fileName : RESOURCES) {
-          try (InputStream is = getClass().getResourceAsStream(TESTNG_RESOURCE_PREFIX + fileName)) {
+          try (InputStream is = load(fileName)) {
             if (is == null) {
               throw new AssertionError("Couldn't find resource: " + fileName);
             }
-            Files.copyFile(is, new File(m_outputDirectory, fileName));
+            Files.copyFile(is, new File(outputDirectory, fileName));
           }
         }
         all = Files.readFile(header);
-        Utils.writeUtf8File(m_outputDirectory, "index.html", xsb, all);
+        Utils.writeUtf8File(outputDirectory, "index.html", xsb, all);
       }
     } catch (IOException e) {
       Logger.getLogger(Main.class).error(e.getMessage(), e);
     }
   }
+
+  private InputStream load(String fileName) {
+    InputStream is = getClass().getResourceAsStream(Main.TESTNG_RESOURCE_PREFIX + fileName);
+    if (is == null) {
```

Comment:
Proposition:

```java
String path;
if (fileName.equals("jquery.min.js") {
  path = "/META-INF/resources/webjars/jquery/3.5.1/jquery.min.js";
} else {
  path = Main.TESTNG_RESOURCE_PREFIX + fileName;
}
return getClass().getResourceAsStream(path);
```

Reply:
It will allow us to manage the path strategy and keep the default fallback.

Reply:
fixed